analyzer/src/main/kotlin/managers/utils/PythonInspector.kt

```diff
@@ -173,6 +178,85 @@ internal object PythonInspector : CommandLineTool {
 
 private const val TYPE = "PyPI"
 
+internal fun PythonInspector.Result.toOrtProject(
+    managerName: String,
+    analysisRoot: File,
+    definitionFile: File
+): Project {
+    val id = resolveIdentifier(managerName, analysisRoot, definitionFile)
+
+    val setupProject = projects.find { it.path.endsWith("/setup.py") }
```

Comment:
I believe that a `setup.py` in the analysis root won't be found.

Note: Same code exists multiple times below.

Reply:
It's not looking at the root because it uses `endsWith`. Also, the python-inspector output uses absolute paths so it would not be the analysis root but the file system root.

Reply:
Consider dropping the assumption about absolute paths anyway,e.g.: `it.path.substringAfterLast('/') == "setup.py"`.